demo/molecule/dropdownOption/demo/MixedDropdownOptions/index.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/prop-types, no-unused-vars, no-console */
+
+import React, {Component, Fragment} from 'react'
+import MoleculeDropdownOption from '../../../../../components/molecule/dropdownOption/src'
+
+const BASE_CLASS_DEMO = 'DemoMoleculeDropdownOption'
+const CLASS_DEMO_OPTION = `${BASE_CLASS_DEMO}-option`
+
+class MixedDropdownOptions extends Component {
```

Comment:
have you seen this anywhere? please remove it if there is no visual of it. As far as I know these options are not meant to be mixed as one is for multiselection and the other is for single selection and this example might lead to confusion

Reply:
done!